Docs: Add info and links about our hosted services to our grafana install docs (#38544)
grafana/grafana: docs/sources/installation/rpm.md

| Grafana Version           | Package            | Repository                                         |
| ------------------------- | ------------------ | -------------------------------------------------- |
| Grafana Enterprise        | grafana-enterprise | `https://packages.grafana.com/enterprise/rpm`      |
| Grafana Enterprise (Beta) | grafana-enterprise | `https://packages.grafana.com/enterprise/rpm-beta` |
| Grafana OSS               | grafana            | `https://packages.grafana.com/oss/rpm`             |
| Grafana OSS (Beta)        | grafana            | `https://packages.grafana.com/oss/rpm-beta`        |

> > The Enterprise Edition is the default and recommended edition. It includes all the features of the OSS Edition, can be used for free and can be upgraded to the [full Enterprise feature set](https://grafana.com/products/enterprise/?utm_source=grafana-install-page), including support for [Enterprise plugins](https://grafana.com/grafana/plugins/?enterprise=1&utcm_source=grafana-install-page).

Reviewer: ```suggestion
>**Note:** The Enterprise is the recommended and default option. It is available for free and includes all the features of the OSS Edition. You can also upgrade to the [ full Enterprise feature set](https://grafana.com/products/enterprise/?utm_source=grafana-install-page) and has support for [Enterprise plugins](https://grafana.com/grafana/plugins/?enterprise=1&utcm_source=grafana-install-page).
```
